src/skin/popup.html

```diff
@@ -90,14 +90,17 @@ <h1 id="share_title" class="i18n_share_title overlay_title"></h1>
   </div>
 
 <div id='privacyBadgerHeader'>
-  <a id="options" title="i18n_popup_options_button" href='/skin/options.html' class="control-button grey-button tooltip" style="display:block" target="_blank"><img src="/icons/options.svg" alt="i18n_popup_options_button"></a>
-  <a id="share" title="i18n_popup_share_button" class="control-button grey-button tooltip" style="display:block" target="_blank"><img width="25" height="auto" src="/icons/share.svg" alt="i18n_popup_share_button"></a>
-  <a id="help" title="i18n_popup_help_button" href='/skin/firstRun.html' class="control-button grey-button tooltip" style="display:block" target="_blank"><img src="/icons/help.svg" alt="i18n_popup_help_button"></a>
+  <a id="options" title="i18n_popup_options_button" href='/skin/options.html' class="control-button grey-button tooltip" style="display:block" target="_blank"><img width="25" src="/icons/black-options-icon.svg" alt="i18n_popup_options_button"></a>
+  <a id="share" title="i18n_popup_share_button" class="control-button grey-button tooltip" style="display:block" target="_blank"><img width="25" height="auto" src="/icons/black-share-icon.svg" alt="i18n_popup_share_button"></a>
+  <a id="help" title="i18n_popup_help_button" href='/skin/firstRun.html' class="control-button grey-button tooltip" style="display:block" target="_blank"><img width="25" src="/icons/black-help-icon.svg" alt="i18n_popup_help_button"></a>
   <div id='title'>
-    <img src="/icons/badger-48.png" srcset="/icons/badger-128.png 2x" width="48" alt=""> <h2><span class="i18n_name"></span></h2>
+    <img src="/icons/PrivacyBadger-noborder.svg" srcset="/icons/PrivacyBadger-noborder.svg 2x" width="48" alt="">
+    <div id='header-image-stack'>
+      <img id='header-red-eff-logo' src="./images/EFF-red.svg" srcset="./images/EFF-red.svg" width="48" alt=""> <br>
+      <img id='privacy-badger-text-logo' src="./images/PrivacyBadger-ChunkFive-Black.svg" srcset="./images/PrivacyBadger-ChunkFive-Black.svg"  width="180">
```

Comment:
We should instead style the localized name of Privacy Badger (`<span class="i18n_name"></span>`) with CSS. For examples, see how we use custom fonts on the new user welcome page.

Reply:
Good call -- I'm bringing this ask around to the design team to see if they've got this font handy. If so, I'll import it and do the changes to the header text with css instead of with an img tag

Reply:
We already have regular Chunk in the [fonts folder](https://github.com/EFForg/privacybadger/tree/master/src/skin/fonts). As per [credits in our license file](https://github.com/EFForg/privacybadger/blob/2213448632d7a6a44c5af0bb5d1fa2c96a91f908/LICENSE#L90) you should be able to get the bold version from https://www.theleagueofmoveabletype.com/.

Reply:
Just got the bold version from design -- incorporating now, as well as some minor fixing on the new badger logo

Reply:
Still needs doing.

Reply:
Thought I fixed that in a previous commit, it's been fixed in the most recent one